rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.77.0"
+channel = "1.78.0"
```

Comment:
Btw, better to revert this to have zero impact on the runtime/client build.

We'll bump it together with the uplift soonish.

Reply:
yes, it was a fix tentative, it's reverted now